Gemfile

```diff
@@ -1,5 +1,7 @@
 source 'https://rubygems.org'
 
+ruby "~> #{RUBY_VERSION}" if ENV["TRAVIS"]
```

Comment:
I think that only matters if rvm is trying to parse the ruby version from the Gemfile.

If it's causing local problems, we can probably just obfuscate it a bit so rvm doesn't see it at all (e.g., make it a `send`).